app/assets/javascripts/search.es6

```diff
@@ -17,11 +17,12 @@ function setupSearchDropdown(formId) {
 
     const allCategoriesString = $('.homepage-search').attr('data-categories');
     const allCategories = allCategoriesString ? allCategoriesString.match(/[^",\[\]]+/g) : [];
```

Comment:
```suggestion
    const allCategories = (allCategoriesString !== "[]") ? allCategoriesString.match(/[^",\[\]]+/g) : [];
```

Reply:
looks like I introduced a bug with my initial code here, when `allCategoriesString` returns `[]` it causes an issue when trying to run the regex on the next line. I didn't realize because Categories was never empty, but now with the addition of Innovations that can be empty the issue has revealed itself.. 

Upon initial testing it looks like this small change will act as a guard and prevents errors.